### PR TITLE
feat: rename class fd-bar--cosy to fd-bar--cozy

### PR DESCRIPTION
**Pull request by droshev**
## Related Issue
fixes: #1114 

Renamed class `fd-bar--cosy` to `fd-bar--cozy`
Affected components:

- bar.md
- calendar.md
- dialog.md
- menu.md
- message-box.md
- popover.md
- switch.md
- combobox-input.md
- date-picker.md
- multi-input.md
- time-picker.md

### CODE REVIEW

File: stories/message-box/message-box.stories.js [hunk 1]
```diff
@@ -426,7 +426,7 @@ To enable responsive behavior for the paddings use the following modifier classe
 - \`.fd-message-box__content--l\`: 2rem - min-width: 1024px and max-width: 1439px
 - \`.fd-message-box__content--xl\`: 3rem - min-width: 1440px
 
-On mobile devices the component should be in \`cosy\` mode. Add the \`fd-bar--cosy\` modifier class to the header and the footer. The buttons in the footer should also be in Cosy mode. The deafult mode for Buttons is \`cosy\` so no modifier classes are required.
+On mobile devices the component should be in \`cosy\` mode. Add the \`fd-bar--cozy\` modifier class to the header and the footer. The buttons in the footer should also be in Cosy mode. The deafult mode for Buttons is \`cosy\` so no modifier classes are required.
```
Comment (stefanoScalzo) [inline stories/message-box/message-box.stories.js:429]: should be in \`cozy\` mode

File: stories/bar/bar.stories.js [hunk 2]
```diff
@@ -126,7 +126,7 @@ export const tabletAndMobile = () => `
 tabletAndMobile.parameters = {
     docs: {
         iframeHeight: 100,
-        storyDescription: 'For Tablet and Mobile (Cosy mode) apply the <code>fd-bar--cosy</code> class to the container element.'
+        storyDescription: 'For Tablet and Mobile (Cosy mode) apply the <code>fd-bar--cozy</code> class to the container element.'
```
Comment (stefanoScalzo) [inline stories/bar/bar.stories.js:129]: (Cozy mode)

File: docs/pages/components/message-box.md [hunk 3]
```diff
@@ -325,15 +325,15 @@ To enable responsive behavior for the paddings use the following modifier classe
     <li><code>.fd-message-box__content--xl</code>: 3rem - min-width: 1440px</li>
 </ul>
 <br><br>
-On mobile devices the component should be in `cosy` mode. Add the `fd-bar--cosy` modifier class to the header and the footer. The buttons in the footer should also be in Cosy mode. The deafult mode for Buttons is `cosy` so no modifier classes are required.
+On mobile devices the component should be in `cosy` mode. Add the `fd-bar--cozy` modifier class to the header and the footer. The buttons in the footer should also be in Cosy mode. The deafult mode for Buttons is `cosy` so no modifier classes are required.
```
Comment (stefanoScalzo) [inline docs/pages/components/message-box.md:328]: replace all cosy with cozy, not just class name

File: stories/message-box/message-box.stories.js (hunk 1 above)
Comment (droshev) [reply stories/message-box/message-box.stories.js]: done

File: docs/pages/components/message-box.md (hunk 3 above)
Comment (droshev) [reply docs/pages/components/message-box.md]: done

File: stories/bar/bar.stories.js (hunk 2 above)
Comment (droshev) [reply stories/bar/bar.stories.js]: done

**Pull request merged**